src/Form/input-classifiers.test.js

```diff
@@ -155,3 +112,26 @@ describe('Input Classifiers', () => {
         })
     })
 })
+
+describe('Real-world form tests', () => {
+    const testCases = require('./test-cases/index')
+
+    test.each(testCases)('Test %s fields', (caseName, form, done) => {
+        document.body.innerHTML = form
+        // When we require autofill, the script scores the fields in the DOM
+        require('../autofill.js')
+
+        // A human classified these fields, so we want to make sure the script matches them
+        const manuallyScoredFields = document.querySelectorAll('[data-manual-scoring]')
```

Comment:
The expectations is that the human tester marks these fields with `data-manual-scoring=someExpectedValue` and we then check that the script gave the same value.